components/desktop/partials/KnitImportantWidgetItem.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div v-if="show" class="knit-important-widget-item">
+    <span @click="hideMessage" class="knit-important-widget-item__message-hider"> x </span>
```

Comment:
Use `button` or `a` instead of `span'. `Span` tag is non-focusable. Semantically this button links to the next page too.
Using `x` letter for close button is very weird solution :p Please use an icon font. 
You can use simple `__close-button` instead of `hider`, but this is only personal :)